docusaurus/docs/testing/extensions-test.md

```diff
@@ -0,0 +1,32 @@
+# Extensions Tests
+
+The Extensions framework is tested by the GitHub action `check-plugins.yaml`. This runs the script `./shell/scripts/test-plugins-build.sh`
+which runs the actual tests.
+
+The tests check the following:
+
+- Ensures that Verdaccio is installed and running locally (Verdaccio provides a local NPM registry that the tests can use to publish packages without having to publish to npm)
```

Comment:
During the test it requires me credentials, should it be ficticious or which one should I use?
Is it possible (for sure is in CI) to have env vars for this?

When I try with random values I get this errror
```
npm ERR! code E401
npm ERR! Unable to authenticate, your authentication token seems to be invalid.
npm ERR! To correct this please trying logging in again with:
npm ERR!     npm login

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/gleo/.npm/_logs/2023-04-18T08_40_15_742Z-debug.log
error Command failed with exit code 1.
```

Is it meant to be configured in somehow?

Reply:
I've seen that we have some credentials in the code, which for some reasons are not passed to the command.
When I use them 2 of the 3 login pass, but then it fails on `cd pkg/rancher-components && npm publish` which should be bypassed by Verdaccio:
<img width="734" alt="Screenshot 2023-04-19 at 17 13 38" src="https://user-images.githubusercontent.com/5009481/233122855-75c33f6e-b32d-4674-a7aa-f5d887a2344e.png">
<img width="721" alt="Screenshot 2023-04-19 at 17 13 47" src="https://user-images.githubusercontent.com/5009481/233122867-1a75f0bb-2607-45da-836a-dbd985e929e1.png">
<img width="718" alt="Screenshot 2023-04-19 at 17 24 43" src="https://user-images.githubusercontent.com/5009481/233124063-7d6ab342-0910-423d-a098-980825c8cb1b.png">

Reply:
Here's another screenshot where the packages are loaded correctly and I can login as admin (still have to do it manually in CLI) but the final step fails.
I think we can probably use this screenshot as well to display what should happen.

![Screenshot 2023-04-20 at 09 50 34](https://user-images.githubusercontent.com/5009481/233297790-86862872-116a-4ffc-95c0-4899cef25867.png)

Reply:
To publish a second time you need to change the version, so it may be worthy to mention it somewhere.

Reply:
Changing `npm` command to `yarn` and add the registry seems solve the last step:

```
    "publish:lib": "cd pkg/rancher-components && yarn publish"
```

@nwmac how is it possible that this is different between local and CI?